### PR TITLE
fix(DB/creature_loot_template) Increase droprate for item Healthy Dragon Scale

### PR DESCRIPTION
**Pull request by Zoidwaffle**
## Changes Proposed:
-  Increase droprate of [Healthy Dragon Scale](https://classic.wowhead.com/item=13920/healthy-dragon-scale) from 6% to 20%

## Issues Addressed:
This is affecting quest [Healthy Dragon Scale](https://classic.wowhead.com/quest=5582/healthy-dragon-scale) which is only available once you finish the [Plagued Hatchlings Quest](https://classic.wowhead.com/quest=5529/plagued-hatchlings).

- The AC database has a droprate setting of 6%
- The TC database has a droprate of 10%
- These guys went for 100% : https://wotlk-twinhead.twinstar.cz/?npc=10678
- Here the rate is set to ~21% : https://classic.wowhead.com/item=13920/healthy-dragon-scale 
- The comments indicate that the drop rate was low in vanilla - 3-5 scales for a whole run.
- A comment from TBC mentions they had 12 scales in a run.

So this is all RNG and so on and so forth, but 6% is probably a bit too low while 100% is also too much.

**My compromise is 20% unless anyone knows a more accurate rate!**

## Tests Performed:
- Applied SQL, killed dragons and counted scales

## How to Test the Changes:

- `.quest add 5529`
- `.quest complete 5529`
- `.quest reward 5529`
- `.go c id 10678`
- kill and loot hatchlings - since this is RNG, you might not see a change every time

## Known Issues and TODO List:

- [ ]
- [ ] 

## Target Branch(es):
- [x] Master

 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

### CODE REVIEW

**Review by FrancescoBorzi — APPROVED** [review]

**Comment by FrancescoBorzi** [comment]
setting it as [To be merged] unless someone has proof of a more specific drop rate. Otherwise, 20% sounds a good compromise given the circumstances.

**Comment by pklloveyou** [comment]
Tested

File: data/sql/updates/pending_db_world/rev_1612869459845259064.sql [hunk 1]
```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1612869459845259064');
+
+-- Increase drop rate of Healthy Dragon Scale
+
+UPDATE `creature_loot_template` SET `Chance`=20 WHERE `Item` = 13920;
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1612869459845259064.sql:5]:
```suggestion
UPDATE `creature_loot_template` SET `Chance`=20 WHERE `Item` = 13920 AND `reference` = 0;
```

**Comment by Kitzunu** [comment]
I just want to make something clear when it comes to data like this.

https://wotlk-twinhead.twinstar.cz/?npc=10678 and anything like it are shit sources, as you have understood to your comment above :+1:  

https://classic.wowhead.com/item=13920/healthy-dragon-scale looking just straight at the drop rate on wowhead is also shit, because it is never accurate. Their addon can be used on private servers and live servers, and still then it wouldn't give an accurate show, but looking at comments and working your way out from there is good :) 

Otherwise, TC opted for 10, you opt for 20. Is there a real way of knowing? No.
good job

**Comment by Zoidwaffle** [comment]
Cheers @Kitzunu - yeah, I know that data sources can be tainted and stuff like twinhead is an unstrustworthy reference at best... And I have no issue what so ever in lowering chance to 10 or just leaving it at 6... It just seemed a bit on the low side once I started looking at it. Thanks for the feedback - and if we all get buried in damned dragon scales next time in Scholomance, we turn the knob down a bit again :heart:

**Pull request merged**